frontend/src/components/maps/leaflet/InfoSlideOut/InfoContent.tsx

```diff
@@ -84,6 +97,21 @@ const getHeading = (propertyTypeId: PropertyTypes | null) => {
   }
 };
 
+const displayProjectStatus = (workflowCode: string) => {
+  switch (workflowCode) {
+    case Workflows.ERP:
+      return 'Property is in Enhanced Referral Process';
+    case Workflows.SPL:
+      return 'Property is on the Surplus Properties List';
+    case Workflows.ASSESS_EX_DISPOSAL:
+      return 'Property has been disposed externally';
```

Comment:
This is incorrect.  This workflow means the project has been approved for exemption from ERP.

Reply:
`Property has been approved for exemption from ERP`

Reply:
ah okay, the comment in the enum is incorrect then . I'll make changes